server/paymentProviders/stripe/virtual-cards.js

```diff
@@ -121,6 +119,24 @@ const setCardStatus = async (virtualCard, status = 'canceled' | 'active' | 'inac
   return data;
 };
 
+const recordInActivityLog = async (virtualCard, amount, currency) => {
+  const CollectiveId = virtualCard.collective.id;
+  const collective = await models.Collective.findByPk(CollectiveId);
+  const user = await models.User.findByPk(virtualCard.UserId);
+  const responsibleAdmin = await models.Collective.findByPk(user.CollectiveId);
```

Comment:
I think we can extract user and collective from virtual card here.
```suggestion
  const collective = virtualCard.collective;
  const user = virtualCard.user;
  const responsibleAdmin = await models.Collective.findByPk(user.CollectiveId);
```